rbac/management/workspace/view.py

```diff
@@ -129,21 +136,46 @@ def update(self, request, *args, **kwargs):
         """Update a workspace."""
         return super().update(request, *args, **kwargs)
 
+    @pgtransaction.atomic(isolation_level=pgtransaction.SERIALIZABLE)
+    def _move_atomic(self, request, *args, **kwargs):
+        target_workspace_id = self._parent_id_query_param_validation(request)
+        target_workspace = get_object_or_404(Workspace, id=target_workspace_id, tenant=request.tenant)
```

Comment:
See if we can remove this (model not needed to be retrieved here, can rely on 403 or 400 level error from other checks)

Reply:
Done 👍